### PR TITLE
add error catches

### PR DESCRIPTION
**Pull request by nathanweatherly**
Signed-off-by: Nathan Weatherly <nweather@redhat.com>

### CODE REVIEW

File: e2e/framework/scheme.go [hunk 1]
```diff
@@ -6,13 +6,29 @@ import (
...
+	scheme   = runtime.NewScheme()
+	setupLog = ctrl.Log.WithName("setup")
+)
 
 func init() {
-	clusterv1.AddToScheme(scheme)
-	addonv1alpha1.AddToScheme(scheme)
-	k8sscheme.AddToScheme(scheme)
-	authenticationv1alpha1.AddToScheme(scheme)
+	err := clusterv1.AddToScheme(scheme)
```
Comment (yue9944882) [inline e2e/framework/scheme.go:18]: nit: conventionally we do one liner error assertion by `utilruntime.Must(..)` for the scheme registration

Comment (nathanweatherly) [reply e2e/framework/scheme.go:18]: ah-- like this? 
```
utilruntime.Must(clientgoscheme.AddToScheme(scheme))
```

Comment (nathanweatherly) [reply e2e/framework/scheme.go]: much prettier! fixed!

**Review by yue9944882 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**